tools/src/main/scala/scala/scalanative/build/Discover.scala

```diff
@@ -19,6 +19,9 @@ object Discover {
   def optimize(): Boolean =
     getenv("SCALANATIVE_OPTIMIZE").forall(_.toBoolean)
 
+  def multithreadingSupport(): Boolean =
+    getenv("SCALANATIVE_MULTITHREADING").exists(_.toBoolean)
```

Comment:
I don't think we should introduce support for *more* environment variables. Users should configure this exclusively through `nativeConfig`. We have some env variables for compatibility purposes, but we shouldn't add new ones.

Reply:
I agree with this - I plan to make changes that would allow the user to use any env var they like when initializing NativeConfig.